peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+PEP: 772
+Title: Packaging governance process
+Author: Barry Warsaw <barry@python.org>,
+        Deb Nicholson <deb@python.org>,
+        Pradyun Gedam <pradyunsg@gmail.com>
+Status: Draft
+Type: Process
+Topic: Governance, Packaging
+Created: 21-Jan-2025
+Replaces: 609
+
+
+========
+Abstract
+========
+
+This PEP proposes a Python Packaging Council with broad authority over
+packaging standards, tools, and implementations. Like the Python Steering
+Council, they seek to exercise this authority as rarely as possible; instead,
+they use this power to establish standard processes.
+
+Like :pep:`8016`, instead of trying to do everything in one PEP, this PEP
+focuses on providing a minimal-but-solid foundation for further governance
+decisions.
+
+========================
+Motivation and Rationale
+========================
+
+As Python packaging has matured, several interrelated problems with the current
+way of managing the technical development, decision making and processes have
+become apparent.
+
+----
+PyPA
+----
+
+The `Python Packaging Authority (PyPA)`_ was created to take over the
+maintenance of `pip`_ and `virtualenv`_ from Ian Bicking, led by Brian Rosner,
+Carl Meyer and Jannis Leidel. Over the years, additional projects got added to
+the PyPA and the tooling maintained by the PyPA became the baseline tooling for
+Python packaging, with additional package ecosystems using and building on top
+of the tools and interoperability standards maintained by the PyPA.
+
+:pep:`609` formalised PyPA's authority over the existing packaging tools, the
+interoperability standards and their operation as a group of independent
+projects under the same umbrella that aim to be interoperable. It also stated
+that PyPA should be expected to provide opinions, insight and experience when
+ecosystem-wide changes are being proposed. In particular, the work to support
+existing projects and maintain a user guide has been excellent.
+
+At the time of writing, PyPA is still less formal than its counterpart on the
+CPython side. It does not have a community elected body and individuals within
```

Comment:
What is the "counterpart" referring to here?

Reply:
PyPA <-> CPython core + steering council, essentially.

Reply:
> remove reference here to CPython

Did this here.